docs/3-entry-points/README.md

```diff
@@ -276,15 +265,16 @@ Transfer `_amount` tokens from `_from`'s lock by `msg.sender` to `_to`
   - Checks that contract has been initialized
   - Check that owner’s lock is enough
   - Checks that amount input is not zero
+  // Note: for these next three functions that involve amounts, shouldn't we also have pre-conditions for checking the amount is inside a bound?
```

Comment:
Just wanted to sanity check, since we explicitly state it elsewhere, should we declare checks on the amounts / balances as preconditions here?